scripts/max-loader.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const async = require('async');
+const fs = require('fs');
+const path = require('path');
+const request = require('request');
+const requestProgress = require('request-progress');
+const mkdirp = require('mkdirp');
+const cp = require('child_process');
+const CronJob = require('cron').CronJob;
+
+var target = path.join(process.cwd(), '../db');
+
+// create db folder
+mkdirp.sync(target);
+
+log('Downloading to %s', target);
+
+// import the list of files to download
+var sources = require('../sources.json');
+var work = [];
+
+// push each file-load-function onto the work queue
+for (var source in sources) {
+  var url = sources[source];
+  log('adding %s', url);
+  work.push(download(url, path.join(target, source)));
+}
+
+function download(url, target) {
+  // closure to separate multiple file-download
+  return function downloadFunctor(callback) {
+    var stream = requestProgress(request(url));
+    stream.pipe(fs.createWriteStream(target)).on('finish', function () {
+      // lets copy the existing file, renaming it to
+      // target-backup (without .gz extension)
+      var existing_file = target.slice(0, -3);
+      var existing_file_backup = existing_file + '-backup';
+      cp.execFile('cp', [existing_file, existing_file_backup]);
+      // force overwrite, even if file exists already
+      // since we took a backup, this wont cause an issue (hopefully)
+      cp.execFile('gunzip', [target, '-f']);
+      callback();
+    });
+  };
+}
+
+function log() {
+  console.log.apply(console, arguments);
+}
+
+function startDownload(work) {
+  log('Last Update: ', new Date());
+  async.parallel(work, function (err) {
+    if (err) {
+      return log(err);
+    } else {
+      log('Download complete');
+    }
+  });
+}
+
+// start download when script is invoked
+startDownload(work);
+
+// after that run periodically
+new CronJob('30 30 1 * * 3', function() {
```

Comment:
instead of `magical` CronJob update,  the module should provide an `.update()` (or better named) call, that either takes options or that uses options provided in the constructor. then we can:

1. call `.update()` on server start to download the zip if geolocation feature is enabled and file is not present.
2. create a `.js` (update-locale.js) script that calls `.update()` in it and updates sources.